glusterd2/volgen/xlators.go

```diff
@@ -0,0 +1,162 @@
+package volgen
+
+import (
+	"path"
+	"strings"
+
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/glusterd2/xlator"
+	"github.com/gluster/glusterd2/pkg/utils"
+)
+
+func (xl *Xlator) isEnabled(volinfo *volume.Volinfo, tmplName string) bool {
+	// If Volinfo.Options can contains xlator, check for xlator
+	// existence in the following order: FullName, Name and Suffix
+	// For example changelog can be enabled using
+	// `brick.features.changelog` or `features.changelog`
+	// or `changelog`
+	// volinfo can be nil in case of cluster level graphs
+	if volinfo != nil {
+		value, exists := volinfo.Options[xl.fullName(tmplName)]
+
+		if !exists {
```

Comment:
can we combine this `if` and next `if` statement?

Reply:
`exists` will be overwritten in first statement. Both can't be merged